reference/exercise-concepts/collatz-conjecture.md

```diff
@@ -0,0 +1,33 @@
+# Collatz Conjecture
+
+## Task
+
+The Collatz Conjecture or 3x+1 problem can be summarized as follows:
+
+Take any positive integer n. If n is even, divide n by 2 to get n / 2. If n is odd, multiply n by 3 and add 1 to get 3n + 1. Repeat the process indefinitely. The conjecture states that no matter which number you start with, you will always reach 1 eventually.
+
+Given a number n, return the number of steps required to reach 1.
+
+## Concepts
+
+- .h file: the most common approach is to use a `.h` for the declarations
+- .cpp file: the most common approach is to use a `.cpp` for the definitions
+- include guard: protect against multiple inclusion within a single translation unit
+- .cpp includes .h: a `.cpp` file should include its `.h` file
+- namespaces: the function is member of a namespace
+- functions: used as the main entry point for the exercise
+- including directives: `<stdexcept>` is needed for `std::domain_error`
+- function arguments: starting number
+- return values: the result is returned by the function
+- conditionals using if: conditionally execute logic using an `if` statement
```

Comment:
The [example solution](https://github.com/exercism/cpp/blob/main/exercises/practice/collatz-conjecture/.meta/example.cpp) does not use `if` but a conditional expression.

Reply:
Thanks for reviewing this!
Do you see a problem with the wording? I meant the `if` statement in line 5 to trigger the exception.

Reply:
&hellip;I missed that one.

No problem with the wording, just pointing out possible oversights.

Reply:
So, no problem here, right?

Reply:
Yes.